trunk/SUAVE/Methods/Weights/Buildups/Common/wiring.py

```diff
@@ -5,6 +5,7 @@
 # Created: Jun, 2017, J. Smart
 # Modified: Feb, 2018, J. Smart
 #           Mar 2020, M. Clarke
+#           May 2021, M. Clarke
```

Comment:
Not your problem, but this should be updated for motors not attached to a wing.

Reply:
I agree, I think this method is suitable for winged eVTOL (the correlations Jordan got from the vahana paper)